app/posts/2022-06-07-parent-child-relationships.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Owning organisations and managing agents
+date: 2022-06-07
+author: Jon Roobottom
+description: Allowing for complex organisation relationships
+---
+So far, in this private beta phase, we've moved two organisations over to our new service. They're "simple" organisations in that they own housing stock and don't subcontract the management of that stock to anyone else.
+
+However, most providers who own stock will often engage other organisations to manage stock on their behalf. We've been calling these organisations "managers" or "owner/managers", and the relations between them and the stock owners "parent" and "child"[^1]. 
+
+These relationships can get quite complex. Here's an example of how several organisations might be connected:
+
+* Organisation A owns stock X and can submit logs for their stock.
+* Organisation B owns stock Y and can submit logs of their stock.
+* Organisation C doesn't own stock but can submit logs on behalf of organisations A or B.
+* Organisation D owns stock Z and can submit logs for their stock and on behalf of organisation B.
+* Organisation E doesn't own stock but can submit logs on behalf of organisation D.
+
+To enable more organisations to submit logs, we need to change the service. 
+
+![A diagram showing the complex relationships between different organisations in CORE](organisational-relationships.png "The complex relationships between organisations A, B, C, D, and E")
+
+## Asking users about who owns and manages the property
+The two organisations using our service in private beta own and manage their stock. We, therefore, don't have to ask them about this in the "set up your lettings log" section of the service. We can *infer* the owner and manager as it's always the organisation that the user is attached to.
```

Comment:
```suggestion
The 2 organisations using our service in private beta own and manage their stock. We don't have to ask them about this in the "set up your lettings log" section of the service because we it is always the organisation that the user is attached to.
```